source/common/network/utility.cc

```diff
@@ -52,6 +52,18 @@ Address::InstanceConstSharedPtr Utility::resolveUrl(const std::string& url) {
   }
```

Comment:
We can remove the exception for `Utility::resolveUrl` also. But yes, this can be a separate PR since this isn't your fault.

Reply:
Let's limit the scope of this PR to the issue. utility.cc has other functions that throw Exception, and they deserve a separate PR.